src/screens/Settings.js

```diff
@@ -101,7 +101,7 @@ export class Settings extends React.Component {
             )
           )}
 
-          <HathorList infinity>
+          <HathorList title={t`General Settings`}>
```

Comment:
It applies an "infinity style". In practice it prolongs the list frame to appear as it goes beyond the screen. But we don't want this style in the settings section anymore, because we will have 2 settings lists.